marklogic-data-hub/src/test/java/com/marklogic/hub/hubcentral/conversion/AllStepsReferencedTest.java

```diff
@@ -12,17 +11,16 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
-
 import java.io.File;
 import java.nio.file.Path;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Iterator;
+import java.util.*;
 
 import static org.junit.jupiter.api.Assertions.*;
 
 class AllStepsReferencedTest extends AbstractHubCoreTest {
 
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
There's an objectMapper already on each instance of AbstractHubCoreTest (inherited from TestObject).

Reply:
ok, it wasn't used so I removed it.